week2/homework/src/index.js

```diff
@@ -1,3 +1,95 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+const program = require('commander');
+const {readToDos} = require('./readtodos.js');
+const {writeToDos} = require('./writetodos.js');
+const {resetToDos} = require('./resettodos.js');
+
+const args = process.argv.slice(2);
+const cmd = args[0];
+
+program
+  .version('0.1.0')
+  .parse(process.argv);
+// interface is not complete yet
+
+async function app() {
+  const toDos = await readToDos();
+  switch (cmd) {
+    case 'list':
+      readToDos().then(console.log);
+      break;
+
+    case 'add':
+      const task = args[1];
+      // let toDos = await readToDos();
+      toDos.push({
+        task,
+        done: false
+      });
+      await writeToDos(toDos);
+      const newToDos = await readToDos();
+      console.log(newToDos);
+      break;
+
+    case 'remove':
+      const toRemoveTaskIndex = Number(args[1]) - 1;
+      if (toRemoveTaskIndex >= 0 && toRemoveTaskIndex < toDos.length) {
+        toDos.splice(toRemoveTaskIndex, 1);
+        await writeToDos(toDos);
+        const remainingToDos = await readToDos();
+        console.log(remainingToDos);
+      }
+      else {
+        console.log(`Not a valid entry, to check the list type 'list'`);
+      }
+      break;
+
+    case 'reset':
+      resetToDos()
+        .catch(() => ([]));
```

Comment:
This `catch` is a bit confusing. You are not doing anything with the array that you are returning. If you want to ignore the errors that might be thrown by you `resetToDos` function, return `undefined` or `null` instead of an array, and put a comment to explain what you're doing.